applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -1737,7 +1737,7 @@ void BaseSolidElement::CalculateAndAddKm(
 {
     KRATOS_TRY
 
-    noalias( rLeftHandSideMatrix ) += IntegrationWeight * prod( trans( B ), Matrix(prod(D, B)));
+    MathUtils<double>::BtDBProductOperation(rLeftHandSideMatrix, IntegrationWeight*D, B, false);
```

Comment:
I thought this was already changed, that's why I implemented BtDBProductOperation

Reply:
IDK why in fact.. should be faster... Where did you get the algorithm? or did you think about it?

Reply:
I did it by 
![imagen](https://user-images.githubusercontent.com/19991680/151007440-ff59f6c5-b312-45ab-96ad-4bc868a35268.png)
ZA HANDO